concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1440,14 +1479,14 @@ jobs:
     plan:
       - aggregate:
           - get: pipeline-trigger
-            passed: [ 'cf-deploy']
+            passed: ['post-deploy','availability-tests']
```

Comment:
I want to run the tests to help me tell _what_ the failure was. I see no harm in running them... It is quite possible that even though there _was_ some downtime during the deploy, now the environment is perfectly healthy again - which is a thing good to know.